libs/utils/utils.js

```diff
@@ -428,30 +429,16 @@ export function decorateImageLinks(a) {
   const images = a.querySelectorAll('img');
   if (!images.length) return a;
   [...images].forEach((img) => {
-    // Check if the image has an alt tag with a url & pipe - 'https... |'
-    if (!(hasUrlWithPipe(img.alt))) return a;
-    try {
-      // //Mine for URL and alt text
-      const splitText = img.alt.split('|');
-      const textUrl = new URL(splitText.shift().trim());
-      const altText = splitText.join('|').trim();
-      if (altText) img.alt = altText;
-
-      const pic = img.closest('picture');
-      const picParent = pic.parentElement;
-      const aTag = createTag('a', { href: textUrl, class: 'desc-link' }, pic);
-      if (aTag.href.includes('#_blank')) {
-        aTag.setAttribute('target', '_blank');
-        aTag.href = aTag.href.replace('#_blank', '');
-      }
-      picParent.append(aTag);
-      return a;
-    } catch (e) {
-      /* c8 ignore next 3 */
-      console.log('Failed to decorate image links', e.message);
-      return a;
-    }
+    if (!img.alt || !(hasUrlWithPipe(img.alt))) return;
```

Comment:
Since the default return seems to be the initial `a`, should it also be returned here?

Reply:
This looks right to me since this check is in the `images` loop and just returns if condition isn't met. The function param `a` does get returned at the end of the func. LMK if i'm off on this though. Thanks

Reply:
Yup, you're right, resolving conversation 👍